.github/deploy.sh

```diff
@@ -9,7 +9,6 @@ echo "Making the docs for master"
 mkdir out/master/
 cp util/gh-pages/index.html out/master
 cp util/gh-pages/script.js out/master
-cp util/gh-pages/lints.json out/master
```

Comment:
With https://github.com/rust-lang/rust-clippy/blob/cbf019b413417918ef1a89c9cef7d64c0c65d8e0/.github/workflows/deploy.yml#L48

Would this create an issue with deploying the 1.81 docs? cc @flip1995

Reply:
What issue do you expect?

Reply:
If it uses the `index.html` of the time but the latest `deploy.sh` it would be missing the required `lints.json`

Reply:
Oh I see. An in-between situation.

Reply:
Yeah, we probably have to push old versions of the `html` files to the gh-pages` branch manually for the 2 releases following this. 

Thanks for the ping! This should block improvements here though. I figured something similar out before and am optimistic to be able to do that again :)

Reply:
Just to confirm: did you forget "not" in "This should block improvements"? :fearful: